src/Menu/SideMenus.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import classes from "./SideDrawer.css"
+
+const menus = (props) => {
```

Comment:
Destructure the props and use it. handle the role props in failure scenario too. make sure your page won't break in any scenarios